Source/NSString+HYPDate.h

```diff
@@ -14,9 +14,9 @@ static NSString * const HYPTimeRangeFormat = @"%@ - %@";
 + (NSString *)hyp_dateStringFromDate:(NSDate *)date;
 + (NSString *)hyp_dateStringFromDate:(NSDate *)date withFormat:(NSString *)dateStringFormat;
 
-+ (NSString *)hyp_timeStringFromDateString:(NSString *)dateString;
```

Comment:
I would suggest pretty much the same for most of the methods that take one parameter.